internal/auth/auth.go

```diff
@@ -65,7 +66,7 @@ type Result struct {
 	Domain       string
 	RefreshToken string
 	AccessToken  string
-	ExpiresIn    int64
+	ExpiresAt    time.Time
```

Comment:
Making this change enabled me to reuse the `Result` type for client credential access tokens. The associated arithmetic gets slightly shuffled as a result.